_examples/render/main.go

```diff
@@ -0,0 +1,153 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"math/rand"
+	"net/http"
+	"reflect"
+	"time"
+
+	"golang.org/x/net/context"
+
+	"github.com/pressly/chi"
+	"github.com/pressly/chi/middleware"
+	"github.com/pressly/chi/render"
+)
+
+type RuntimeObject struct {
```

Comment:
in the example, I think it would be nice to use a "real-world" analogy. this way people reading the example can relate to it some kind of domain .. for example, maybe use the idea of Articles and a simple blog presenting other data